csharp/src/Ice/LoggerAdminLogger.cs

```diff
@@ -56,18 +61,33 @@ internal LoggerAdminLogger(Communicator communicator, ILogger localLogger)
         {
             LocalLogger = (localLogger as LoggerAdminLogger)?.LocalLogger ?? localLogger;
             _loggerAdmin = new LoggerAdmin(communicator, this);
-        }
 
-        internal void Log(LogMessage logMessage)
-        {
-            List<LogForwarder>? remoteLoggers = _loggerAdmin.Log(logMessage);
-            if (remoteLoggers != null)
+            // Create an unbounded channel to ensure the messages are sent from a separate thread. We don't allow
+            // synchronous continuations to ensure that writes on the channel are never processed by the writer
+            // thread.
+            _channel = Channel.CreateUnbounded<LogMessage>(new UnboundedChannelOptions {
```

Comment:
I would put the opening `{` on a new line..

Reply:
Ok I didn't know this one. Is it the same for lambda expressions?

Reply:
Yes.